Backend/index.js

```diff
@@ -1,12 +1,11 @@
-const express = require("express");
-const cors = require("cors");
-const AuthRoutes = require("./routes/Auth");
-const session = require("express-session");
-require("dotenv").config();
+import express, { json, urlencoded } from "express";
```

Comment:
According to you eslint config, these and other strings in the file need to be single quotes.